scripts/scripts.js

```diff
@@ -0,0 +1,227 @@
+import * as dom from "./modules/dom-utils.js";
+import * as nf from "./modules/notesFunction.js";
+import * as test from "./modules/test.js";
+import * as gf from "./modules/generalFunctions.js";
+import * as cf from "./modules/calendarFunctions.js";
+
+dom.logAllEle(); // log all initially captured HTML elements
+
+/////////////////////////////////////////
+/* LIVE TIME */
+
+// Function to get current time & display
+function currentTime() {
+  // Get the current date and time
+  const now = new Date();
+
+  // Get the current hour and pad it with leading zeros if necessary
+  const hours = String(now.getHours()).padStart(2, "0");
+
+  // Get the current minute and pad it with leading zeros if necessary
+  const minutes = String(now.getMinutes()).padStart(2, "0");
+
+  // Get the current second and pad it with leading zeros if necessary
+  const seconds = String(now.getSeconds()).padStart(2, "0");
+
+  // Format the time as a string in the "HH:MM:SS" format
+  const time = `${hours}:${minutes}:${seconds}`;
```

Comment:
[toLocaleTimeString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString) this could be done with an exisiting method